en/identity-server/next/docs/apis/organization-apis/restapis/org-application-mgt.yaml

```diff
@@ -0,0 +1,2591 @@
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApplicationPatchModel'
+        description: This represents the application details to be updated.
+        required: true
+      responses:
+        '200':
+          description: Successfully Updated
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'PATCH' \
+            'https://localhost:9443/o/api/server/v1/applications/{application-id}' \
+            -H 'accept: */*' \
+            -H 'Authorization: Bearer {bearer_token}' \
+            -H 'Content-Type: application/json' \
+            -d '{
+            "name": "pickup",
+            "description": "This is the configuration for Pickup application.",
+            "imageUrl": "https://example.com/logo/my-logo.png",
+            "claimConfiguration": {
+              "dialect": "LOCAL",
+              "claimMappings": [
+                {
+                  "applicationClaim": "firstname",
+                  "localClaim": {
+                    "uri": "http://wso2.org/claims/username"
+                  }
+                }
+              ],
+              "requestedClaims": [
+                {
+                  "claim": {
+                    "uri": "http://wso2.org/claims/username"
+                  },
+                  "mandatory": false
+                }
+              ],
+              "subject": {
+                "claim": {
+                  "uri": "http://wso2.org/claims/username"
+                },
+                "includeUserDomain": false,
+                "includeTenantDomain": false,
+                "useMappedLocalSubject": false
+              },
+              "role": {
+                "mappings": [
+                  {
+                    "localRole": "admin",
+                    "applicationRole": "Administrator"
+                  }
+                ],
+                "includeUserDomain": true,
+                "claim": {
+                  "uri": "http://wso2.org/claims/username"
+                }
+              }
+            }
+            }'
+    delete:
+      tags:
+        - Applications
+      summary: |
+        Delete application by ID.
+      operationId: deleteApplication
+      description: |
+        This API provides the capability to delete an application by ID. <br>
```

Comment:
Shall we specify this is possible for non shared apps

Reply:
Done